dist/leaflet.css

```diff
@@ -327,6 +327,25 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	border-bottom-right-radius: 2px;
 	}
 
+
+/* Dark Mode */

```

Comment:
Having the dark mode styles in the main leaflet.css means users will always have to download these extra KB even if the `darkMode` option is set to false. That's something that should be considered. Maybe there should be a separate stylesheet for dark mode styles? I don't have a particularly strong opinion about this, perhaps maintainers do.

Reply:
This could lead to two separate HTTP requests with additional round-trip delay and overhead, so I think it would be preferable to have the both light and dark in the same CSS file.